samcli/commands/delete/command.py

```diff
@@ -63,12 +64,26 @@
     is_flag=True,
     required=False,
 )
+@click.option(
+    "--s3-bucket",
+    help=("The S3 bucket path you want to delete."),
```

Comment:
Let's don't forget to check with docs team before pushing this change.

Reply:
Checked with docs team, all good to go